spec.bs

```diff
@@ -248,6 +262,104 @@ If using a centralized mechanism for fetching keys, user agents should have a re
 
 When using a registration process, it is recommended that user agents apply an expiration date to registration requests, to allow for the removal of deprecated or no longer active issuers.
 
+VOPRF Methods {#voprf-methods}
+====================================
+
+This document encodes protocol messages in the TLS presentation language from section 3 of [[!RFC8446]].
+
+To <dfn>serialize protocol messages</dfn> and  <dfn>deserialize protocol messages</dfn>, protocol messages are encoded and interpreted as described in section 3 of [[!RFC8446]].
+
+For Private State Tokens, the VOPRF protocol is initialized using P-384 for the curve (`G` in the functions described below) and `nonce_size` is defined as 64. 
```

Comment:
Would it be correct to link to https://www.ietf.org/archive/id/draft-irtf-cfrg-voprf-21.html#name-oprfp-384-sha-384 for the curve?

For the nonce_size, it looks like our nonces are twice as big as in https://www.ietf.org/archive/id/draft-robert-privacypass-batched-tokens-01.html#name-client-to-issuer-request? That could also probably use a note.